native-maven-plugin/src/main/java/org/graalvm/buildtools/maven/NativeCompileNoForkMojo.java

```diff
@@ -101,6 +105,12 @@ public void execute() throws MojoExecutionException {
         maybeSetMainClassFromPlugin(this::consumeConfigurationNodeValue, "org.apache.maven.plugins:maven-assembly-plugin", "archive", "manifest", "mainClass");
         maybeSetMainClassFromPlugin(this::consumeConfigurationNodeValue, "org.apache.maven.plugins:maven-jar-plugin", "archive", "manifest", "mainClass");
         maybeAddGeneratedResourcesConfig(buildArgs);
+
+        if (enableSBOM) {
```

Comment:
This should also guard that the EE version of Native Image is used. What's an easy and clean way to differentiate between EE and CE from this context?

Reply:
Implemented this by calling `native-image --version` and checking if the output contains "Oracle GraalVM".

Reply:
This is directly related to the [long standing discussion](https://github.com/graalvm/native-build-tools/issues/357) about toolchain support. We need a _reliable_ way to make a difference between vendors, editions and features included in a release.

Reply:
Superseded by [this thread](https://github.com/graalvm/native-build-tools/pull/623#discussion_r1791567680).